index.rst

```diff
@@ -15,5 +10,241 @@ Change Record
+All significant development stages of each planning package are tracked through *milestones*.
+Each planning package has at least one milestone, used to mark the completion of a given package.
+It is not uncommon to have multiple milestones per package to track progress along the way.
+Milestones can be related to multiple planning packages, potentially from different WBSes.
+Each milestone has a description, and due date; it has no duration, and no resources assigned to it.
+It is not uncommon for a milestone to related to multiple planning packages, possibly from
+different WBSes.
+
+Each activity within a planning package that involves a cross-team dependency is required to end with
+a milestone; this includes activities related to delivering software components, hardware, and services.
+
+Further, milestones have *levels*:
+
+* Level 1 denotes the most important milestones exposed at the NSF level
+
+* Level 2 denotes cross-subsystem milestones (for example, DM milestones that affect the Camera Subsystem)
+
+* Level 3 denotes cross-team milestones within a subsystem (for example, Middleware milestones that effect the DRP Team)
+
+* Level 4 denotes internal milestones within a team.
+
+*O(few hundred)* milestones have been defined to-date to track the progress of DM activities.
+
+Relationship between milestones, as well as between milestones and planning packages are captured:
+milestones typically *block* planning packages, milestones can *block* other milestones, or they
+can *relate* to other milestones.
+
+Baseline Long-Term Plan
+-----------------------
+
+Planning packages, together with milestones form the *Baseline Plan*. The plan is under change control.
+
+The plan is stable. Any change to the plan, including changing scope, cost or schedule, must be approved
+by the appropriate change control body:
+
+* Change Control Board approval for any non-trivial change to a planning package or Level 1 and
+  Level 2 milestone
+
+* DM's TCT approval for Level 3 milestones
+
+* no approval is needed for Level 4 milestones, however any non-trivial change must be communicated
+  to the DM team
+
+* trivial changes, such as small corrections / clarifications to narrative that do not affect
+  scope, time or budget are allowed without approval.
+
+Typically, adjustments to Level 1, 2 and 3 milestones are made every 6 or 12 months. Changes to Level 4
+milestones can occur more frequently.
+
+Short-Term Planning
+-------------------
+
+The short-term plan is managed through *epics* and *stories*, and executed through *cycles*.
+
+In the EVMS world, epics map directly to *activities*, and stories map to *activity steps*.
+
+Epics and Stories
+~~~~~~~~~~~~~~~~~
+
+Typically, planning packages encompass relatively large blocks of work. To plan and execute work
+in details, finer-grain planning is required. This is implemented through epics and stories.
+
+Each epic captures a non-trivial work that is associated with a subset of work defined in a
```

Comment:
"Each epic captures a non-trivial subset of the work defined in a planning package."